scripts/serverless.js

```diff
@@ -46,8 +51,29 @@ process.once('uncaughtException', (error) =>
     serverless,
     hasTelemetryBeenReported,
     commandUsage,
-  })
-);
+  });
+  clearTimeout(keepAliveTimer);
```

Comment:
I would probably add it as first thing in this listener (as that feels more accurate mentally), but technically it won't make  areal difference (crash that may prevent reaching this line, will force process exit anyway)